src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
Make sure that the path to the star-active.svg is correct and accessible from the compiled CSS. If the images directory is not on the same level as the compiled CSS, the path may need to be adjusted.